devel/etc/package.tt

```diff
@@ -54,7 +54,7 @@ A [% name %] should be one of the following types:
 
 has [% name %] => (
     is        => 'rw',
-    predicate => 1,
+    predicate => '_has_[% name %]',
```

Comment:
This will require an "incompatible" change. I'm fine with it, because I suspect not many people are using the predicates. But it should be noted in the Changes.